src/auto/tree_row_reference.rs

```diff
@@ -0,0 +1,71 @@
+// This file was generated by gir (4d68d19) from gir-files (11e0e6d)
+// DO NOT EDIT
+
+use TreeModel;
+use TreePath;
+use ffi;
+use glib;
+use glib::object::IsA;
+use glib::translate::*;
+
+glib_wrapper! {
+    pub struct TreeRowReference(Boxed<ffi::GtkTreeRowReference>);
+
+    match fn {
+        copy => |ptr| ffi::gtk_tree_row_reference_copy(mut_override(ptr)),
+        free => |ptr| ffi::gtk_tree_row_reference_free(ptr),
+    }
+}
+
+impl TreeRowReference {
+
+    pub fn new<T: IsA<TreeModel>>(model: &T, path: &TreePath) -> TreeRowReference {
```

Comment:
Done thanks to [gtk-rs/gir#290](https://github.com/gtk-rs/gir/pull/290). I'm not sure - may I update the gir module for a regen? (last update is six month back)